scripts/qa_crowbarsetup.sh

```diff
@@ -839,7 +840,7 @@ function do_installcrowbar()
     # run in screen to not lose session in the middle when network is reconfigured:
     screen -d -m -L /bin/bash -c "$instcmd ; touch /tmp/chef-ready"
 
-    if [ -n "$wanthyperv" ] ; then
+    if [ -n "$wanthyperv" ] || [[ -n `echo ${hvmix[*]} | grep "hyperv"` ]]; then
```

Comment:
Actually, this is even better:

    if [ -n "$wanthyperv" ] || [[ ${hvmix[*]} == *hyperv* ]]; then